frontend/app/controllers/Joiner.scala

```diff
@@ -170,11 +183,25 @@ trait Joiner extends Controller with ActivityTracking {
       Future.sequence(optFuture.toSeq).map(_.headOption)
     }
 
+    val refererCookie = request.cookies.get("GU_MEM_REFERER").map(_.value.replace(Config.guardianUrl, ""))
```

Comment:
Rather than configure in the Guardian url, we can just use [Uri-parsing](https://github.com/NET-A-PORTER/scala-uri#parsing-uris) to get the 'path' part of the url:

```
request.cookies.get("GU_MEM_REFERER").map(_.value.path)
```